src/components/navigation/navigation-dropdown-button-custom.js

```diff
@@ -1,9 +1,17 @@
 import { css, html, LitElement } from 'lit';
 import { highlightBorderStyles, highlightButtonStyles } from './navigation-styles.js';
 import { DropdownOpenerMixin } from '@brightspace-ui/core/components/dropdown/dropdown-opener-mixin.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
 
 class NavigationDropdownButtonCustom extends DropdownOpenerMixin(LitElement) {
 
+	static get properties() {
+		return {
+			openerLabel: { type: String, attribute: 'opener-label' },
+			openerLabelledby: { type: String, attribute: 'opener-labelledby' }
```

Comment:
I don't think `labelled-by` is going to work here since the elements it references can't span shadow roots.